src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard.module.ts

```diff
@@ -11,6 +11,10 @@ import { CephSharedModule } from '../shared/ceph-shared.module';
 import { CardComponent } from './card/card.component';
 import { DashboardPieComponent } from './dashboard-pie/dashboard-pie.component';
 import { DashboardComponent } from './dashboard/dashboard.component';
+import { MdsSummaryPipe } from './mds-summary.pipe';
+import { MgrSummaryPipe } from './mgr-summary.pipe';
+import { MonSummaryPipe } from './mon-summary.pipe';
+import { OsdSummaryPipe } from './osd-summary.pipe';
```

Comment:
I'd suggest that you import the module where these pipes are defined, instead of performing the individual imports. You can check in other places how these pipes are used.

Reply:
Hey @epuertat could you point to me where these pipes are located?

Reply:
Usually we have all the pipes collectively defined under [pipes] (https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/pipes.module.ts) folder which is `src/app/shared/pipes/` but these four pipes are not under there but located inside the [dashboard ](https://github.com/ceph/ceph/tree/main/src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard) `src/app/ceph/dashboard` folder. Its been there for a long time so I would suggest moving these pipes to pipes module and then import the pipes module into the `dashboard.module.ts` file and then you can use all the pipes.